cas/src/main/java/org/springframework/security/cas/web/CasAuthenticationEntryPoint.java

```diff
@@ -107,7 +107,7 @@ protected String createServiceUrl(final HttpServletRequest request,
 	 * @param serviceUrl the service url that should be included.
 	 * @return the redirect url. CANNOT be NULL.
 	 */
-	protected String createRedirectUrl(final String serviceUrl) {
+	protected String createRedirectUrl(final HttpServletRequest request, final String serviceUrl) {
```

Comment:
We cannot modify the signature of a protected method (it is not passive) without first deprecating and then waiting for a major release. Can you please modify it so that it is passive?